pandas/tests/indexing/test_indexing.py

```diff
@@ -542,6 +571,18 @@ def test_astype_assignment_with_dups(self):
         # result = df.get_dtype_counts().sort_index()
         # expected = Series({'float64': 2, 'object': 1}).sort_index()
 
+    def test_coercion_with_contains(self):
+        # Related to GH 16957
+        # Checking if Int64Index contains np.inf should catch the OverflowError
+        for val in [np.inf, np.nan]:
```

Comment:
We can avoid this `for`-loop using `pytest` parametrization, which you can see <a href="http://doc.pytest.org/en/latest/parametrize.html">here</a>